hw2/03-count.html

```diff
@@ -1,34 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>03 - Count</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="03-count.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label>Highlight all occurences of a word:</label>
-        <input />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
-      </div>
-    </main>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>03 - Count</title>
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+		/>
+		<style>
+			.highlight {
+				background-color: yellow;
+				font-weight: bold;
+			}
+		</style>
+		<script defer src="03-count.js"></script>
+	</head>
+	<body
+		class="d-flex justify-content-center align-content-center p-5 bg-primary"
+	>
+		<main class="bg-white w-25 p-5 m-5 mt-2 rounded">
```

Comment:
The width needs to be wider, at 1024px wide, it does not look right
![image](https://github.com/shrutivd/Frontend-web-tech/assets/59980086/08c6b468-47d7-4a4e-b3d3-3c32197780a3)

Reply:
I think this should look good now

Reply:
I made some more changes. Please let me know if it still is not as expected